objects/obj_enunit/Alarm_1.gml

```diff
@@ -223,9 +223,15 @@ if __b__
 {
 
 repeat(20){j+=1;
-    if (dudes[j]="Imperial Guardsman"){dudes_ac[j]=20;dudes_hp[j]=20;men+=dudes_num[j];}
+    if (dudes[j]="Imperial Guardsman"){
+		scr_en_weapon("Lasgun",true,dudes_num[j],dudes[j],j);
+		scr_en_weapon("Combat Knife",true,dudes_num[j],dudes[j],j);
+		dudes_ac[j]=3;dudes_hp[j]=25;men+=dudes_num[j];}
     
-    if (dudes[j]="Heavy Weapons Team"){scr_en_weapon("Heavy Bolter",true,dudes_num[j],dudes[j],j);dudes_ac[j]=0;dudes_hp[j]=100;men+=dudes_num[j];}
+    if (dudes[j]="Heavy Weapons Team"){
+		scr_en_weapon("Heavy Bolter",true,dudes_num[j],dudes[j],j);
+		scr_en_weapon("Combat Knife",true,dudes_num[j],dudes[j],j);
+		dudes_ac[j]=3;dudes_hp[j]=25;men+=dudes_num[j];}
```

Comment:
Make dudes_hp be 100. Considering it's a team of 4 guys and guardsmen have 25 HP each